cronjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: gpu-stress-test
+spec:
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: resource.gpu
+                    operator: In
+                    values:
+                    - "true"
+          containers:
+          - name: gpu-stress-test
+            image: iperezx/gpu-stress-test:latest 
+            imagePullPolicy: IfNotPresent
+            command:
+            - -m 1
```

Comment:
to really encourage the box to warm up, i would like to run the gpu stress test for atleast 5 minutes every 30 minutes or so.